### ISSUE TITLE
Problem with the pretrained weights

### ISSUE THREAD

**Issue by sinaazar** [yaohungt/Gated-Spatio-Temporal-Energy-Graph#10]
Hi,

I tried to use the i3d implementation here for another task. However, it seems like the provided weights are not just from training on Kinetics. I guess it was also trained on charades.
this is the error when I try to load weights for a model to classify Kinetics (as an example).
`size mismatch for logits.conv3d.weight: copying a param with shape torch.Size([157, 1024, 1, 1, 1]) from checkpoint, the shape in current model is torch.Size([400, 1024, 1, 1, 1])`
Is it possible to include Kinetics-only weights?

Thanks

**Comment by yaohungt** [comment]
There are a lot of i3d pre-trained models for Kinetics online.

**Comment by sinaazar** [comment]
I see. I preferred this implementation.
Anyways, the Readme is misleading. You might want to edit it.

**Comment by yaohungt** [comment]
OK, Thx
